transformer_engine/pytorch/module/base.py

```diff
@@ -448,7 +449,7 @@ def set_activation_dtype(self, inp: torch.Tensor) -> None:
                     f" Found input dtype: {dtype} and {name!r} dtype: {param.dtype}"
                 )
         for name, buf in self.named_buffers():
-            if buf is not None:
+            if buf is not None and ('weight' in name or 'bias' in name):
```

Comment:
Do you mean removing the whole `for name, buf` loop? I didn't find any other buffers either but I thought maybe the loop was there for some other purposes.

The `weight/bias` check would help CI from failing at `onnx_causal_mask` tests, because the its dtype is Bool and q/k/v dtype is usually not (`set_activation_dtype`).

Reply:
I was thinking of removing the loop over `named_buffers`. I believe this check was because we used to store fused QKV weights in registered buffers.